### ISSUE TITLE
Revisions are not being created

### ISSUE THREAD

**Issue by ibonelli** [outline/outline#3544]
**To Reproduce**
1. Create a document
2. Wait for more than 5mins and change it
3. Check the revisions table in the database
4. There will be no new revision for the document

**Expected behavior**
When a document changes (beyond the [debounce time](https://github.com/outline/outline/discussions/2016)) a new revision should be created.

**Screenshots**
I think it is clear the description, but if it helps I don't mind saving a video.

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.62.0 (3ac125d560ece69d71c2d21e76ca1af2a0ac9112)

**Desktop (please complete the following information):**
 - OS: Heroku
 - Browser Chrome
 - Version v99

**Comment by tommoor** [comment]
I don't know what you're missing but there are many thousands of revisions created daily in the cloud hosted instance so it's definitely not broken in this way. You should try upgrading to the latest version first of all

**Comment by ibonelli** [comment]
Hi there, thanks for the quick reply the last time.

So, I updated to latest version ([v0.64.3](https://github.com/outline/outline/releases/tag/v0.64.3)) and I still see no revisions. I checked and they seem to have stopped at a point, but I have no record of which upgrade might have broke it. Maybe there is a new configuration I'm missing? At this time I see nothing on the log that would indicate a deprecation, but I must admit I do not check every time I do an upgrade.

Also looking at the [RevisionsProcessor.ts](https://github.com/outline/outline/blob/main/server/queues/processors/RevisionsProcessor.ts) code I see nothing that would prevent the revisions from being created. The log doesn't report problems.

Help?
